pkg/api/serialization_test.go

```diff
@@ -128,7 +128,8 @@ func TestList(t *testing.T) {
 }
 
 var nonRoundTrippableTypes = sets.NewString()
-var nonInternalRoundTrippableTypes = sets.NewString("List", "ListOptions", "PodExecOptions", "PodAttachOptions")
+
```

Comment:
nit: extra blank line (and all three of these things could be in one `var ( ... )` block...)